README.md

```diff
@@ -54,6 +54,8 @@ Updating the version of the SCIM bridge involves a few steps that are described
 
 See [Updating the SCIM bridge version](./op-scim-bridge/chart/op-scim-bridge-mp/README.md#updating-the-scim-bridge-version) for details.
 
+After that has been done, please ensure `publishedVersion` in [schema.yaml](./op-scim-bridge/schema.yaml) matches the `appVersion` in [Chart.yaml](./op-scim-bridge/chart/op-scim-bridge-mp/Chart.yaml), and that they both match the SCIM bridge docker image version we're updating to.
```

Comment:
```suggestion
Next, ensure the `publishedVersion` in [schema.yaml](./op-scim-bridge/schema.yaml) matches the `appVersion` in [Chart.yaml](./op-scim-bridge/chart/op-scim-bridge-mp/Chart.yaml), and that they both match the SCIM bridge docker image version we're updating to.

Additionally the [Chart.lock](./op-scim-bridge/chart/op-scim-bridge-mp/Chart.lock) `op-scim-bridge` dependency chart version should match that from our updated [SCIM bridge Helm Chart](https://github.com/1Password/op-scim-helm). Note this version will not match the docker image version. 
```

Reply:
The `Chart.lock` note in this suggestion is already implicitly covered in the chart's [readme](https://github.com/1Password/op-scim-gcp-marketplace/tree/alice/v2.9.0/op-scim-bridge/chart/op-scim-bridge-mp#updating-the-scim-bridge-version) in step two, where `helm dependency update .` is done. Since this is a generated file, following the previous steps will result in the correct version numbers in the `Chart.lock` file.